apps/website/src/components/Header.astro

```diff
@@ -45,9 +46,32 @@ const { pathname } = Astro.url;
         ></path>
       </svg>
     </a>
+
+    <button id='search-action-item' class='action-item'>
+      <VisuallyHidden>Search</VisuallyHidden>
+
+      <svg width='32' height='32' viewBox='0 0 24 24' fill='currentColor'>
+        <path
+          d='M11 9.7c.7-1 1.1-2.2 1.1-3.5.1-3.5-2.7-6.2-6-6.2C2.7 0 0 2.7 0 6.1s2.7 6.1 6.1 6.1c1.3 0 2.5-.4 3.5-1.1l4.9 4.9 1.4-1.4zm-5 .5c-2.3 0-4.1-1.8-4.1-4.1S3.7 2 6 2s4.1 1.8 4.1 4.1-1.8 4.1-4.1 4.1'
+        ></path>
+      </svg>
```

Comment:
I was having a bit of trouble centering the search icon (https://github.com/iTwin/iTwinUI-icons/blob/main/packages/icons/icons/search.svg?short_path=dc61d25). Could someone please take a look at this?